src/test/java/bio/terra/common/auth/AuthService.java

```diff
@@ -33,8 +40,10 @@
   private JacksonFactory jsonFactory = JacksonFactory.getDefaultInstance();
   private File pemfile;
   private String saEmail;
-  private Map<String, String> userTokens = new HashMap<>();
-  private Map<String, String> directAccessTokens = new HashMap<>();
+  private Map<String, String> userTokens =
+      Collections.synchronizedMap(new PassiveExpiringMap<>(TimeUnit.MINUTES.toMillis(10)));
```

Comment:
There are a ton of ways to write this, I kind of like `Duration.ofMinutes(10).toMillis()`

Reply:
Ah, that's good to know!